src/applications/personalization/view-dependents/reducers/index.js

```diff
@@ -0,0 +1,37 @@
+import { splitPersons } from '../util';
+
+import {
+  FETCH_ALL_DEPENDENTS_SUCCESS,
+  FETCH_ALL_DEPENDENTS_FAILED,
+} from '../actions';
+
+const initialState = {
+  loading: true, // app starts in loading state
+  error: null,
+  onAwardDependents: null,
+  notOnAwardDependents: null,
+};
+
+let allPeople = null;
+
+function allDependents(state = initialState, action) {
+  switch (action.type) {
+    case FETCH_ALL_DEPENDENTS_SUCCESS:
+      allPeople = splitPersons(action.response.persons);
+      return {
+        ...state,
+        loading: false,
+        onAwardDependents: allPeople.onAward,
+        notOnAwardDependents: allPeople.notOnAward,
+      };
+    case FETCH_ALL_DEPENDENTS_FAILED:
+      return {
+        ...state,
+        allDependents: action,
```

Comment:
Should this return an error that frontend can use? I think it should also set loading to false.

Reply:
Looks like error handling will be added in a future pull request.